sessionctx/variable/sysvar.go

```diff
@@ -683,6 +689,8 @@ const (
 	CharsetDatabase = "character_set_database"
 	// CollationDatabase is the name for collation_database system variable.
 	CollationDatabase = "collation_database"
+	// GroupConcatMaxLen is the name for group_concat_max_len system variable.
+	GroupConcatMaxLen = "group_concat_max_len"
```

Comment:
We'd better not add this constant here. Maybe we can take the session variable `max_chunk_size` as an example? `max_chunk_size` and `group_concat_max_len` are all defined in the `defaultSysVars`

Reply:
you mean `tidb_max_chunk_size` ? It's also defined as a constant in `tidb_vars.go`...

Reply:
The former `character_set_database`, `collation_database` etc  are defined as constants. Maybe it's better to make them consistent here ?  All defined as constants or all not ?